html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <div>Email</div>
+    <label for="email">
+      <input name="email" type="text">
+    </label>
+
+    <label for="coding-cool">
+      <div>Why is coding cool?</div>
+    </label>
+    <select name="coding-cool" id="coding-cool">
+      <option value="">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+
+    <div>Favorite Language (choose one)</div>
+    <label for="html">
+      <input name="fave-lang" id="html" type="radio" value="html" checked="checked">HTML
+    </label>
+    <label for="css">
+      <input name="fave-lang" id="css" type="radio" value="css">CSS
+    </label>
+    <label for="javascript">
+      <input name="fave-lang" id="javascript" type="radio" value="javascript">JavaScript
+    </label>
```

Comment:
For radio buttons to work properly they need to all share the same `name` attribute value. This will make it so only one radio can be selected at a time.